### DISCUSSION TITLE
Drawer menu design changes while on production URGENT PLEASE

### DISCUSSION THREAD

**Discussion by nimeshmaharjan1** [saadeghi/daisyui#1965] [Q&A]
Edit: its not the drawer just the menu, I even updated daisyui to ver 3 and also added it as a dev dependency you can find the ss below

This is the production build which is deployed on vercel

![image](https://github.com/saadeghi/daisyui/assets/86953478/306c9012-4519-4a35-8aca-7ba6babca789)

and this is the localhost I cant seem to find the issue thats causing this?

![image](https://github.com/saadeghi/daisyui/assets/86953478/1d3f8728-8082-4abb-b1f2-3f2e65949ef1)

![image](https://github.com/saadeghi/daisyui/assets/86953478/e1b66b18-9c12-4561-b402-9e7a28be8c58)

This is the code

![image](https://github.com/saadeghi/daisyui/assets/86953478/095489cd-260e-4754-b8da-66f645412baf)

**Comment by saadeghi** [comment]
Can you please share a minimal reproduction repo so I can run the code and investigate the issue? Because it looks like your production code is purging some styles.

It could be a config issue. It could be a build tool bug. It could be CSS scoping issue. It could be CSS minifier config issue. It's really hard to help with screenshots

**Comment by nimeshmaharjan1** [comment]
Thank you for replying 
This is the repo https://github.com/nimeshmaharjan1/ems
Branch: prod

I have used a workaround with flex and some styling instead of using the menu for now

**Reply by saadeghi** [reply]
>I have used a workaround with flex and some styling instead of using the menu for now

So, should I search your repo and guess where is the problem? 😅  
Please provide a "minimal reproduction repo" my friend.
